flight/core/Dispatcher.php

```diff
@@ -78,7 +78,7 @@ public function run(string $name, array $params = [])
      * Assigns a callback to an event.
      *
      * @param string $name Event name
-     * @param Closure(): (void|mixed) $callback Callback function
+     * @param Closure(): (void|mixed)|callable $callback Callback function
```

Comment:
callable and Closure(): (void|mixed) are same... I use Closure because it allows any valid callable and LSP-intelephense can detect that in `$result = $myParamClosure()`, $result'll be `@var void|mixed`...

And with simple `callable` doc typehint, $result'll be detected as `@var mixed`

Reply:
So I literally did nothing but running phpstan made this rule fail. So I added the little callable line and then it passed.